libraries/matchers.rb

```diff
@@ -32,7 +32,7 @@
     oneview_fcoe_network:               standard_actions + scope_actions + %i[reset_connection_template],
     oneview_firmware:                   %i[add remove create_custom_spp],
     oneview_hypervisor_manager:         standard_actions + %i[update_registration],
-    oneview_hypervisor_cluster_profile: standard_actions + %i[update],
+    oneview_hypervisor_cluster_profile: standard_actions + %i[update delete],
```

Comment:
Remove delete because delete is a part of standard actions.

Reply:
done